packages/commonwealth/client/scripts/utils/avatarUtils.ts

```diff
@@ -0,0 +1,20 @@
+import avatar1 from './../../assets/img/avatars/default-avatar1.png';
+import avatar2 from './../../assets/img/avatars/default-avatar2.png';
+import avatar3 from './../../assets/img/avatars/default-avatar3.png';
+import avatar4 from './../../assets/img/avatars/default-avatar4.png';
+import avatar5 from './../../assets/img/avatars/default-avatar5.png';
+import avatar6 from './../../assets/img/avatars/default-avatar6.png';
+
+const defaultAvatars: string[] = [
+  avatar1,
+  avatar2,
+  avatar3,
+  avatar4,
+  avatar5,
+  avatar6,
+];
```

Comment:
I think we should upload this to s3 and then save it in user profile as they'd get used later on, reference

> Additional Context: when we send data to Connect routes for email notifications  #9415, we should send along the preset image profile (assuming the user has not set a profile

from https://github.com/hicommonwealth/commonwealth/issues/9506

Reply:
@mzparacha @salman-neslit Messaged Sachin about this, waiting to hear back. I say wait until we get a definitive answer from Product before doing more work on this. I assume using an s3 will be his answer, but I think it's safer to wait and see.

Reply:
@salman-neslit @mzparacha Just heard from Sachin, the way that this should function is that they should persist unless the user changes the profile image. So, assigned at random and then persisting unless the user changes it

Reply:
Thanks @Israellund, so @salman-neslit you can upload the new image presets once to s3 (prob look into [uploadImage](https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/client/scripts/views/components/component_kit/cw_cover_image_uploader.tsx#L137) code and upload via that call) and store the urls in the util, then update user profile with that randomly selected avatar url.